Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -215,36 +215,40 @@ section OnFinset
 
 variable [Zero M]
 
+private irreducible_def onFinset_support (s : Finset α) (f : α → M) : Finset α :=
```

Comment:
Is the kernel problematic here or just the elaborator?

Reply:
Thanks, I captured that in the description.

Reply:
That's not a 12% speedup but a 12% slowdown.